Dockerfile

```diff
@@ -155,6 +162,7 @@ LABEL maintainer="Max Goltzsche <max.goltzsche@gmail.com>"
 RUN apk add --no-cache tzdata ca-certificates
 COPY --from=conmon /conmon/bin/conmon /usr/local/lib/podman/conmon
 COPY --from=podman /usr/local/lib/podman/rootlessport /usr/local/lib/podman/rootlessport
+COPY --from=podman /usr/local/libexec/podman/quadlet /usr/local/libexec/podman/quadlet
```

Comment:
It would be good to add the `quadlet` binary to the `podmanall` (last) stage only so that it doesn't end up bloating the minimal image variant but is added to the default one only that contains all binaries.

Reply:
Done.